### PR TITLE
[risk=low][no ticket] Bump some e2e packages

### PR DESCRIPTION
**Pull request by jmthibault79**
Upgrade the `@types` and `jest` packages

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: e2e/package.json [hunk 1]
```diff
@@ -45,14 +45,13 @@
...
-    "@types/faker": "^5.5.6",
-    "@types/fs-extra": "^9.0.11",
-    "@types/jest": "^27.0.2",
-    "@types/jest-environment-puppeteer": "^4.4.0",
-    "@types/lodash": "^4.14.170",
-    "@types/node": "^15.12.5",
-    "@types/rimraf": "^3.0.0",
+    "@types/expect-puppeteer": "^4.4.7",
+    "@types/fs-extra": "^9.0.13",
+    "@types/jest": "^27.5.1",
```
Comment (jmthibault79) [inline e2e/package.json:50]: this is the latest in 27.x - does not upgrade to the new v28

File: e2e/package.json [hunk 2]
```diff
@@ -45,14 +45,13 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@types/expect-puppeteer": "^4.4.5",
-    "@types/faker": "^5.5.6",
```
[the remark this answers is not in the document]
Comment (aweng98) [reply e2e/package.json:49]: Does `faker` needs upgrade? because last time I checked, it didn't have @types build in.

**Review by aweng98 — APPROVED** [review]
👍🏻

File: e2e/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmthibault79) [reply e2e/package.json:49]: you're right!  restored.

(looking into it further, the `faker` author is a troubled individual and his words should perhaps should not be trusted!)

**Pull request merged**